.clang-format

```
@@ -1,37 +1,33 @@
 ---
+Language: Cpp
+# (Options are sorted alphabetically)
 AlignAfterOpenBracket: Align
-AlignConsecutiveAssignments: 'false'
-AlignConsecutiveDeclarations: 'false'
-AlignEscapedNewlinesLeft: 'true'
-AllowAllParametersOfDeclarationOnNextLine: 'true'
-AllowShortBlocksOnASingleLine: 'false'
-AllowShortCaseLabelsOnASingleLine: 'false'
+AlignEscapedNewlines: Right
+AlignOperands: AlignAfterOperator
+AllowAllArgumentsOnNextLine: false
+AllowAllParametersOfDeclarationOnNextLine: false
+AllowShortBlocksOnASingleLine: Empty
+AllowShortCaseLabelsOnASingleLine: true
 AllowShortFunctionsOnASingleLine: None
-AllowShortIfStatementsOnASingleLine: 'false'
-AllowShortLoopsOnASingleLine: 'false'
-AlwaysBreakAfterReturnType: All
-BinPackArguments: 'false'
-BinPackParameters: 'false'
-BreakBeforeBraces: Linux
-ColumnLimit: '80'
-ContinuationIndentWidth: '3'
-IndentCaseLabels: 'false'
-IndentWidth: '3'
-KeepEmptyLinesAtTheStartOfBlocks: 'false'
-Language: Cpp
-MaxEmptyLinesToKeep: '2'
-SortIncludes: 'false'
-SpaceAfterCStyleCast: 'true'
-SpaceBeforeAssignmentOperators: 'true'
-SpaceBeforeParens: Always
-SpaceInEmptyParentheses: 'false'
-SpacesInAngles: 'false'
-SpacesInCStyleCastParentheses: 'false'
-SpacesInContainerLiterals: 'true'
-SpacesInParentheses: 'false'
-SpacesInSquareBrackets: 'false'
-Standard: Cpp11
-TabWidth: '3'
+AllowShortIfStatementsOnASingleLine: Never
+AllowShortLoopsOnASingleLine: false
+BinPackArguments: false
+BinPackParameters: false
+BreakBeforeBinaryOperators: NonAssignment
+BreakBeforeBraces: Attach
+ColumnLimit: 120
+ContinuationIndentWidth: 4
+IndentCaseLabels: false
+IndentWidth: 4
+InsertBraces: true
+KeepEmptyLinesAtTheStartOfBlocks: false
+PointerAlignment: Right
+QualifierAlignment: Custom
+QualifierOrder: ["static", "inline", "const", "volatile", "type", "restrict"]
+SeparateDefinitionBlocks: Always
+SpaceBeforeAssignmentOperators: true
+SpaceBeforeParens: ControlStatements
```

```suggestion
SpaceBeforeAssignmentOperators: true
SpaceBeforeParens: Always
```
I still want to make sure we want to change from `SpaceBeforeParens: Always` (status quo) to `SpaceBeforeParens: ControlStatements`. Re-posting a comment here to invite further opinions, but I do not consider this suggestion a blocker.

---

Yes, this change was made based on the poll